openapi-maven-plugin/src/test/java/io/github/kbuntrock/SpringClassAnalyserTest.java

```diff
@@ -708,4 +708,33 @@ public void name_collision() throws MojoFailureException, IOException, MojoExecu
 		checkGenerationResult("ut/SpringClassAnalyserTest/name_collision.yml", generated.get(0));
 	}
 
+	@Test
+	public void tag_name_collision() throws MojoFailureException, IOException, MojoExecutionException {
+
+		final DocumentationMojo mojo = createBasicMojo(
+			io.github.kbuntrock.resources.endpoint.namecollision.one.MyController.class.getCanonicalName(),
+			io.github.kbuntrock.resources.endpoint.namecollision.two.MyController.class.getCanonicalName());
+
+		final List<File> generated = mojo.documentProject();
+		checkGenerationResult("ut/SpringClassAnalyserTest/tag_name_collision.yml", generated.get(0));
+	}
+
+	@Test
+	public void enpoint_path_collision() throws MojoFailureException, MojoExecutionException {
+
+		final DocumentationMojo mojo = createBasicMojo(
+			io.github.kbuntrock.resources.endpoint.namecollision.three.MyController.class.getCanonicalName());
+
+		MojoRuntimeException exception = null;
+		try {
+			mojo.documentProject();
```

Comment:
Please use assertj to check for exceptions https://www.baeldung.com/assertj-exception-assertion#1-using-assertthatthrownby

Reply:
Good call